app/src/main/java/com/nervousfish/nervousfish/data_objects/RSAKey.java

```diff
@@ -48,36 +52,12 @@ public RSAKey(final Map<String, String> map) throws IllegalArgumentException {
         }
     }
 
-    /**
-     * Get the exponent of an {@link RSAKey}.
-     *
-     * @param key An {@link RSAKey}.
-     * @return The value for the exponent attribute of the {@code key}.
-     * @throws Exception If the provided {@link IKey} is not a {@link RSAKey}.
-     */
-    public static String getExponent(final IKey key) throws IllegalArgumentException {
-        final String type = key.getType();
-        if (!type.equals(ConstantKeywords.RSA_KEY)) {
-            throw new IllegalArgumentException();
-        }
-
-        return ((RSAKey) key).exponent;
+    public String getExponent() {
```

Comment:
I think this is a very bad idea since we want to work with the `IKey` interface as much as possible. The static methods allows you to more easily get the exponent/modules once you know the `IKey` is a `RSAKey`, so you don't have to manually cast etc.